src/coord/src/catalog/builtin.rs

```diff
@@ -1017,6 +1017,16 @@ lazy_static! {
             .with_column("schema_id", ScalarType::Int64.nullable(false))
             .with_column("name", ScalarType::String.nullable(false))
     };
+    pub static ref MZ_CONNECTORS: BuiltinTable = BuiltinTable {
+        name: "mz_connectors",
+        schema: MZ_CATALOG_SCHEMA,
+        desc: RelationDesc::empty()
+        .with_column("id", ScalarType::String.nullable(false))
```

Comment:
nit: I believe these should be indented a level

Reply:
Agreed, it is really surprising that cargo fmt is not doing that in fact.